### DISCUSSION TITLE
Bug? Triple click on headset started to be responded as 2+1

### DISCUSSION THREAD

**Discussion by mars4science** [PaulWoitaschek/Voice#1746] [Q&A]
Triple click on headset started to be responded as 2+1 (forward + pause/resume). I've not changed the app, triple rewinded and noted later triple click started to result in 2+1. Restart did not solve the issue. I've tried web search and then to look at source code based on found:
https://stackoverflow.com/questions/20751493/headset-button-double-click-in-android
https://stackoverflow.com/questions/39694867/android-handle-headset-buttons-events-and-send-information-to-mainactivity

But could not understand how headset events processing in done in the app for sure. Is it single event for triple click (so it seems the issue outside the app) or is it 3 events and somehow app does not process them correctly in some cases (so it looks as a bug)? 

Voice 7.0.3. TIA

**Comment by PaulWoitaschek** [comment]
And what would you expect on a triple click?

**Reply by mars4science** [reply]
@PaulWoitaschek , it skipped backward on triple
